### PR TITLE
EBF-210: Progress Step

### PR DESCRIPTION
**Pull request by moysa**
- Add a new Progress::Stepper component
- Add acceptance tests for Progress::Stepper

### CODE REVIEW

File: addon/components/progress/stepper/index.hbs [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<div class="progress-step-holder">
+  <div class="container">
+    <div class="progress-step-header">
+      <p class="progress-step-title">{{@title}}</p>
+        <LinkTo
+            class="progress-step-close"
+            @route={{this.router.currentRouteName}}
```
Comment (obyknovenius) [inline addon/components/progress/stepper/index.hbs:7]: When we explicitly provide `@currentRoute`, shouldn't we take it into account? Something like
```hbs
@route={{if @currentRoute @currentRoute this.router.currentRouteName}}
```

File: addon/components/progress/stepper/step.hbs [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<div name={{concat "step-" @index}} class={{concat "progress-item " this.state}}>
```
Comment (obyknovenius) [inline addon/components/progress/stepper/step.hbs:1]: What is the purpose of the `name` attribute? Does `<div>` even has it?

Comment (obyknovenius) [inline addon/components/progress/stepper/step.hbs:1]: You don't need `concat`. `class="progress-item {{this.state}}"` will work just as well.

Comment (obyknovenius) [reply addon/components/progress/stepper/step.hbs:1]: If we just need to distinguish these tags from the others, maybe it's better to prefix it with `data-`? Like described [here](https://stackoverflow.com/questions/4962070/attribute-name-not-allowed-on-element-div-at-this-point).

**Review by obyknovenius — APPROVED** [review]

File: addon/components/progress/stepper/index.hbs (hunk 1 above)
Comment (moysa) [reply addon/components/progress/stepper/index.hbs:7]: This is temporary, until designers change this to a button. Currently this property is ignored, but needs to be there for the `LinkTo` component to work properly

File: addon/components/progress/stepper/step.hbs (hunk 2 above)
Comment (moysa) [reply addon/components/progress/stepper/step.hbs:1]: Good point

File: addon/components/progress/stepper/index.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+      return 'active';
+    }
+    else if (index < activeIndex) {
+      return 'completed';
+    }
+
+    return '';
+  }
+
+  @action async confirmCancel(e) {
```
[the remark this answers is not in the document]
Comment (moysa) [reply addon/components/progress/stepper/index.js:33]: Yup, forgot to remove it, thanks

**Pull request merged**